draft-ietf-core-dns-over-coap.md

```diff
@@ -182,9 +182,14 @@ or automatic configuration, e.g., using a CoRE resource directory
 {{-ddr}}.
 Automatic configuration SHOULD only be done from a trusted source.
 
-Support for SVCB Resource Records {{-svcb}}, {{-svcb-dns}} or DNR Service Parameters {{-dnr}}
-are not specified in this document.
-{{-core-dnr}} explores solutions for CoAP for these mechanisms.
+A DoC server can also be discovered using SVCB Resource Records {{-svcb}}, {{-svcb-dns}} or DNR
+Service Parameters {{-dnr}}.
+\[TBD: draft-lenders-core-coap-dtls-svcb\] provides solutions
+to discover CoAP over (D)TLS servers using the "alpn" SvcParam. This document specifies "docpath" as
+a single-valued SvcParamKey whose value (in both presentation and wire format) MUST be a URI
+reference ({{-uri}}) in its path-abempty form. The DoC client MUST send any DoC request to the CoAP
```

Comment:
Editorial fine point: The MUST probably needs to be qualified by "To use the service binding, ..." (the possession of it does not mean that you will be using it).

Reply:
Addressed in https://github.com/core-wg/draft-dns-over-coap/pull/28/commits/1cc69527a9c3094b169eaa2420b39417acba3961